e2e-polybft/helpers_test.go

```diff
@@ -77,3 +80,48 @@ func getRootchainValidators(relayer txrelayer.TxRelayer, checkpointManagerAddr,
 
 	return validators, nil
 }
+
+func hasValidatorSealed(srv *framework.TestServer, firstblock, lastblock uint64, validators polybft.AccountSet, validatorAddr ethgo.Address) (bool, error) {
```

Comment:
As this refers to the `srv` couldn't this be included as func of `TestServer`?

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/280ec4c1235e74c2e4e08d3b527915863bfc3d85